engine/launcher/src/main/scala/org/enso/launcher/components/LauncherRunner.scala

```diff
@@ -191,4 +192,54 @@ class LauncherRunner(
       )
     }
   }
+
+  private val uploadIntroducedVersion: SemVer =
+    SemVer(0, 2, 16, Some("SNAPSHOT"))
```

Comment:
I feel like this should be pulled out to a constant rather than just left inline here. The version is also wrong. `0.2.16` was released today.

Reply:
To clarify, a constant in a `Constants.scala` or something.

Reply:
This version was written here before the release, indeed I should update it.

Where is this `Constants` file located? Or shall I create a new one in the launcher?

Reply:
Just create a new one in the launcher!